### DISCUSSION TITLE
Why this doesnt work with amd gpu?

### DISCUSSION THREAD

**Discussion by vidalperezbohoyo** [leggedrobotics/darknet_ros#317] [General]
Hello! Im new in this. I dont have idea why only this works with cuda nvidia. All IA only works with nvidia?
Why is not compatible proccess this with my radeon?

**Comment by FPSychotic** [comment]
I guess radeon is compatible with OpenCL which is a equivalent of CUDA for the non nVidia cards.
I would check OpenVINO and onnx.
Maybe in Radeon site explain something how they integrate their products for this area. 
I'm not very sure really, just trying you get a fast response and something to look at.

**Comment by tomlankhorst** [comment]
It's because lots of GPU computing is built on CUDA which is a proprietary NVidia platform. 
Other companies offer alternatives to CUDA like OpenCL but NVidia has had a big head start.
I agree it would be nice to have compatibility with multiple GPU vendors but that would require a big overhaul of the codebase while it's likely easier to start from scratch.
